plugins/modules/eos_lacp_interfaces.py

```diff
@@ -37,8 +37,7 @@
 }
 
 DOCUMENTATION = """module: eos_lacp_interfaces
-short_description: Manage Link Aggregation Control Protocol (LACP) attributes of interfaces
-  on Arista EOS devices.
+short_description: Lacp_interfaces resource module.
```

Comment:
the way I have noticed in other platform modules it should be like "LACP interfaces resource module"